.github/workflows/lint-and-test.yml

```diff
@@ -72,10 +72,11 @@ jobs:
         run: |
           cargo check -p proof-of-sql --no-default-features --features="std"
           cargo check -p proof-of-sql --all-targets --no-default-features --features="std"
-      - name: Run cargo check (proof-of-sql-parser) with no_std target.
+      - name: Run cargo check (proof-of-sql-parser & proof-of-sql) with no_std target.
         run: |
           rustup target add thumbv7em-none-eabi
-          cargo check --target thumbv7em-none-eabi --no-default-features
+          cargo check -p proof-of-sql-parser --target thumbv7em-none-eabi --no-default-features
+          cargo check -p proof-of-sql --target thumbv7em-none-eabi --no-default-features
```

Comment:
Can you create a GitHub issue that adds a wasm target for this crate.
We can add it after the initial dev work.
It's not a big rush, so it could be good for an external contributor/bounty.

Reply:
Sure! https://github.com/spaceandtimelabs/sxt-proof-of-sql/issues/597